webui/react/package.json

```diff
@@ -40,6 +40,7 @@
     "screenfull": "^5.0.2",
     "smoothscroll-polyfill": "^0.4.4",
     "sprintf-js": "^1.1.2",
+    "stylelint-scss": "^3.18.0",
```

Comment:
**chore:** move this to dev-dependencies?

Reply:
I used --dev . looks like that deprecation warning was not messing around